charts/pmm/templates/ingress.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if .Values.ingress.enabled -}}
+{{- $serviceName := .Values.service.name -}}
+{{- $ingressPathType := .Values.ingress.pathType -}}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "pmm.fullname" . }}
+  labels:
+    {{- include "pmm.labels" . | nindent 4 }}
+  annotations:
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+{{- with .Values.ingress.annotations }}
+{{ toYaml . | indent 4 }}
+{{- end }}
+spec:
+{{- if .Values.ingress.ingressClassName }}
+  ingressClassName: {{ .Values.ingress.ingressClassName | quote }}
+{{- end }}
+{{- if .Values.ingress.tls }}
+  tls:
+  {{- range .Values.ingress.tls }}
+    - hosts:
+      {{- range .hosts }}
+        - {{ . | quote }}
+      {{- end }}
+      secretName: {{ .secretName }}
+  {{- end }}
+{{- end }}
+  rules:
+  {{- range .Values.ingress.hosts }}
+    - host: {{ .host | quote }}
+      http:
+        paths:
+        {{- range .paths }}
+          - path: {{ . }}
+            pathType: {{ $ingressPathType }}
+            backend:
+              service:
+                name: {{ $serviceName }}
+                port: 
+                  number: 443
```

Comment:
no problem, 80 or 443 or an array? 

I see we have both ports on the app spec.

https://github.com/Kencove/percona-helm-charts/blob/main/charts/pmm/templates/statefulset.yaml#L84

will need mapped in:

https://github.com/percona/percona-helm-charts/blob/main/charts/pmm/templates/service.yaml#L17

```
  ports:
    - port: 443
      targetPort: https
      protocol: TCP
      name: https
```
you could just expose both on the service but otherwise, need 
[] of port and targetPort

https://kubernetes.github.io/ingress-nginx/user-guide/tls/#ssl-passthrough
you can do some type of passthrough. But then seems to be implementation specific. there would need to be some assumption of using the first or a helper needed to pick on > 1.  So I pushed the array with the `use first item` for ingress approach. example is 80 but can leave both, or either one to work. ( don't ask me for the ci test though! ) 

it all my apps we just use http on the app and let the cert-manager manage all the ingress tls. This does work fine with tls as is and with the new options.